README.md

```diff
@@ -43,17 +43,45 @@ Invalid:
 ```typescript
 import * as pulumi from "@pulumi/pulumi";
 
-const outputInstance: pulumi.OutputInstance<number> = pulumi.output([1, 2, 3])[0];
-const someInterpolatedString = `${outputInstance}`
+const outputInstance: pulumi.OutputInstance<number> = pulumi.output([
+  1, 2, 3,
+])[0];
+const someInterpolatedString = `${outputInstance}`;
 ```
 
 Valid:
 
 ```typescript
 import * as pulumi from "@pulumi/pulumi";
 
-const outputInstance: pulumi.OutputInstance<number> = pulumi.output([1, 2, 3])[0];
-const someInterpolatedString = pulumi.interpolate`${outputInstance}`
+const outputInstance: pulumi.OutputInstance<number> = pulumi.output([
+  1, 2, 3,
+])[0];
+const someInterpolatedString = pulumi.interpolate`${outputInstance}`;
+```
+
+### no-input-in-template-literal
+
+This rule is very similar to `no-output-in-template-literal`, and is intended to prevent objects of type `pulumi.Input<T>` from appearing in template strings.
```

Comment:
```suggestion
Simmilar `no-output-in-template-literal`, this rule guards against using  objects of type `pulumi.Input<T>` from appearing in template strings. (Since `pulumi.Input<T>` can wrap `pulumi.Output<T>` and thus encounter the same error unless you use `pulumi.interpolate`.) 
```